src/components/AboutPage.js

```diff
@@ -1,15 +1,140 @@
+          community to support each other in challenging times. Racial 
+          inequality has long been an issue in the tech space. While change 
+          does not happen overnight, we want to make an active effort to bring 
+          the tech community together to help support Black lives and fight for 
+          racial justice.
+        </p>
+      </Col>
+    </Row>
+
+    <hr />
+    
+    <Row>
+      <Col className="col"></Col>
+      <Col className="col-10">
+        <h2>Community Partners</h2>
+        <p>
+          These are organizations that have helped build, endorse, and promote 
+          the platoform.
+        </p>
+        <ul>
+          <li>HackBeanpot</li>
+          <li>...</li>
+          <li>...</li>
+          <li>...</li>
+          <li>...</li>
+          <li>...</li>
+          <li>...</li>
+          <li>...</li>
+          <li>...</li>
+        </ul>
+      </Col>
+    </Row>
+
+    <hr />
+    
+    <Row>
+      <Col className="col"></Col>
+      <Col className="col-8">
+        <h2>Request a Project</h2>
+        <p>
+          Are you a black business owner, activist, protest organizer, or 
+          community advocate looking for help building a website or online 
+          platform, creating social media content, data visualizations, or other
+           projects?
+        </p>
+        <Col>
+          <LinkContainer to="/project-request">
+            <Button className="button" variant="warning" size="med">
+              Request a Project
+            </Button>
+          </LinkContainer>
+        </Col>
+      </Col>
+    </Row>
+
+    <hr />
+
+    <Row>
+      <Col className="col"></Col>
+      <Col className="col-10">
+        <h2>Work on a Project</h2>
+        <p>
+          Are you a designer, programmer, engineer, data scientist, writer, web 
+          developer, or marketing specialist looking to use your skills to do 
+          pro bono work to support Black lives?
+        </p>
+        <Col>
+          <LinkContainer to="/projects">
+            <Button className="button" variant="warning" size="med">
+              Work on a Project
+            </Button>
+          </LinkContainer>
+        </Col>
+      </Col>
+    </Row>
+
+    <hr />
+
+    <Row>
+      <Col className="col"></Col>
+      <Col className="col-8">
+        <h2>Contact Us</h2>
+        <a href="mailto:buildforblacklives@hackbeanpot.com">
+          buildforblacklives@hackbeanpot.com
+        </a>
+        <br />
+        <a href="https://www.facebook.com/hackbeanpot/">
+          Facebook
+        </a>
+        <br />
+        <a href="https://www.instagram.com/hackbeanpot/">
+          Instagram
+        </a>
+        <br />
+        <a href="https://www.linkedin.com/company/hackbeanpot-inc">
+          LinkedIn
+        </a>
+        <br />
+        <a href="https://twitter.com/hackbeanpot">
+          Twitter
+        </a>
+        {/* TODO include HBP socials */}
+      </Col>
```

Comment:
This would look better with a mail icon: email. 
And then a single row of the other socials can just be the logos as links.
(Total: 2 rows)

Reply:
waiting on decision for icon pack to use